docs/integration/social-networks.md

```diff
@@ -8,28 +8,34 @@ You can set up which social network logins are enabled on which domain in `socia
 
 For every social network, you need `app_id` and `app_secret`. Below you have information on where and how to get them. The credentials then need to be set to the corresponding environment variables.
 
+Every social network require URL for backward redirect to you app.
+In Shopsys platform, you have to add url `{eshop_domain}/social-network/login/{type}` where `{eshop_domain}` is your domain and `{type}` is social network name, e.g.: `google`, `facebook`, `seznam`, etc.
+For local development, social networks usually require https.
+
 ### Facebook
 
 -   go to site https://developers.facebook.com/ and log into Facebook account
 -   create [application](https://developers.facebook.com/apps)
 -   in that application, in `App settings` → `Basic` you will find `app_id` and `app_secret`
 
+P.S.: If you are owner of the Facebook application, and you log in with this owner account, email will not come after login, more info: https://github.com/hybridauth/hybridauth/issues/1372#issuecomment-1542447803
+
 ### Google
 
 -   log into Google account and create [OAuth 2.0 Client IDs](https://console.cloud.google.com/apis/credentials)
+    -   first, you need to create a project,
+    -   then you have to need to click on "+ create credentials" and select "OAuth client ID",
+    -   then, click on "configure consent screen" and fill in the required fields (choose an "internal" user type).
+    -   After you configure the consent screen, once again click on "+ create credentials" and select "OAuth client ID".
+    -   As an application type, choose "web application" and fill in the required fields.
+    -   You need to add an authorized redirect URI here, which is `{eshop_domain}/social-network/login/google`
```

Comment:
Maybe we should emphasizes the requirements for the multi-domain shop? is it necessary to add all eshop domains as redirect URI?

Reply:
That is a good point, I have added it to the general info in 6d9234a33c

Also, I am going to modify our apps to allow all the domains :pray: